plugins/woocommerce/includes/wc-update-functions.php

```diff
@@ -2852,3 +2854,21 @@ function wc_update_930_migrate_user_meta_for_launch_your_store_tour() {
 		)
 	);
 }
+
+/**
+ * Recreate FTS index if it already exists, so that phone number can be added to the index.
+ */
+function wc_update_940_add_phone_to_order_address_fts_index(): void {
+	$fts_already_exists = get_option( CustomOrdersTableController::HPOS_FTS_ADDRESS_INDEX_CREATED_OPTION ) === 'yes';
+	if ( ! $fts_already_exists ) {
+		return;
+	}
+
+	$hpos_controller = wc_get_container()->get( CustomOrdersTableController::class );
+	$result          = $hpos_controller->recreate_order_address_fts_index();
```

Comment:
Should this be done as a background process, instead of synchronously during upgrade? What would be the impact of running this on 100k+ orders?

Reply:
Sync functions already run in the background via the action scheduler, and shouldn't be part of the sync upgrade process.